core/src/main/java/fr/sncf/osrd/standalone_sim/result/ResultTrain.java

```diff
@@ -89,7 +89,6 @@ public SpacingRequirement(String zone, double beginTime, double endTime, boolean
             assert !Double.isNaN(beginTime);
             assert !Double.isNaN(endTime);
             assert Double.isFinite(beginTime);
-            assert Double.isFinite(endTime);
```

Comment:
is it because of partial requirements?

Reply:
It's for tests, we often want to input spacing requirements that have no end duration